src/components/Footer/Footer.js

```diff
@@ -1,7 +1,120 @@
 import React from 'react';
+import css from './Footer.module.scss';
+import left from './image/left.png';
+import right from './image/right.png';
+import { FaFacebook, FaInstagram, FaRegCopyright } from 'react-icons/fa';
+import ch from './image/ch.png';
+import kcp from './image/kcp.png';
 
 function Footer() {
-  return <div>Footer</div>;
+  return (
+    <>
+      <div className={css.banner}>
+        <img className={css.left_banner} alt="left_banner" src={left} />
+      </div>
+      <div className={css.banner}>
+        <img className={css.right_banner} alt="right_banner" src={right} />
+      </div>
+      <div className={css.second_position}>
+        <div className={css.use_info}>
+          <span className={css.fontUse}> 이용안내 </span>
+          <div className={css.cumsu}>
+            검수기준
+      </div>
+          <div className={css.use_policy}>
+            이용정책
+      </div>
+          <div className={css.panalty}>
+            패널티정책
+      </div>
+          <div className={css.community}>
+            커뮤니티 가이드 라인
+      </div>
+        </div>
+        <div className={css.support}>
+          <span className={css.fontUse}>고객지원</span>
+          <div className={css.kongji}>
+            공지사항
+      </div>
+          <div className={css.service}>
+            서비스소개
+      </div>
+          <div className={css.show_room}>
+            쇼룸 안내
+      </div>
+          <div className={css.seller}>
+            판매자 방문접수
+      </div>
+        </div>
+        <div className={css.center}>
+          <span className={css.fontUse}> 고객센터 1588 - 7813 </span>
+          <div className={css.run_time}>
+            운영시간 평일 11:00 ~ 18:00 (토,일 공휴일 휴무)
+      </div>
+          <div className={css.run_time1}>
+            점심시간 평일 13:00 - 14:00
+      </div>
+          <div className={css.run_time2}>
+            1:1 문의하기는 앱에서만 가능합니다.
+      </div>
+          <div className={css.button_q}>
+            <button className={css.bt}>자주묻는 질문</button>
+          </div>
+        </div>
+      </div>
+      <div className={css.bottom_line}>
+      </div>
+      <div className={css.introduce}>
+        회사소개
+    <div className={css.recu}>
+          인재채용
+    </div>
+        <div className={css.susset}>
+          제휴제안
+    </div>
+        <div className={css.use_law}>
+          이용약관
+    </div>
+        <div className={css.priver}>
+          <span className={css.fontUse}>개인정보처리방침</span>
```

Comment:
```suggestion
          <span className={css.font_use}>개인정보처리방침</span>
```
이렇게 수정해주시면 좋을 것 같아요!